telegram-bot/meetup.go

```diff
@@ -93,9 +93,13 @@ func addmeetup(ID int64, msgtext string, client mongo.Client) {
 		if err != nil {
 			log.Fatalln(err)
 		}
+		location, err := time.LoadLocation("Asia/Kolkata")
+		if err != nil {
+			panic(err)
```

Comment:
I don't think panic is preferred. Instead `log.Print(err)` and send a bot message. We are planning to do it in #43 but as well as start here only :).

cc @sidntrivedi012